### DISCUSSION TITLE
Alpha 15

### DISCUSSION THREAD

**Discussion by cameronwhite** [powertab/powertabeditor#333] [General]
### Installing

- For Windows and Mac users, download the installer from the `Assets` section at the bottom of the release notes.
- For Ubuntu users, this release can be installed from the [stable builds PPA](https://launchpad.net/~powertab-developers/+archive/ubuntu/powertabeditor-stable).

### Added
- Many behavior changes to improve the workflow of editing symbols such as tempo markers (#23, #192, #220).
  - These items can now be clicked on and selected, and Delete can then be pressed to remove the selected item.
  - Double-clicking an item will now launch the edit dialog. Previously, it was required to remove and then re-create the item.
- New preferences option to enable playing notes as they are entered (#47).
- New menu item `Help` -> `App Info` which contains information about the installation. This can be useful to include when reporting bugs (#329)

### Changed
- On macOS, the `Delete` key can now be used to remove notes rather than having to press `Fn+Delete`.
- The MIDI device is now initialized in the background, which should avoid some lag the first time playback is activated (#47).
- Qt 5.10 or higher is now required.

### Fixed
- Fixed a MIDI playback bug where adjacent repeated sections resulted in an empty bar being played (#289).
- Fixed a bug where the score's existing subtitle did not show up when opening the file information dialog.
- Fixed a bug where multi-bar rests were not drawn as a filled rectangle.
- Fixed a bug where the pan controls in the mixer had no effect (#276).
- Fixed a rare error when importing .ptb files (#332).

### Credits
Thanks to the following contributors who worked on this release:
- @cameronwhite
- @psyomn

<hr /><em>This discussion was created from the release <a href='https://github.com/powertab/powertabeditor/releases/tag/2.0.0-alpha15'>Alpha 15</a>.</em>

**Comment by Ebdim9th** [comment]
sweet, the notes even play back to duration.....

**Reply by cameronwhite** [reply]
Yep! It tries to match how the note will actually sound

**Reply by Ebdim9th** [reply]
i like it... i was playing around with a transcription I'm doing (which of course will remain only on my computer in private) of Time Waits For Noone by White Wolf...... so far I've got it matched more or less with the recording...

**Comment by BigHonkinDaddy** [comment]
Good release... So much better to be able edit text lyrics (rather than delete the lyrics and having to retype them) - same for many other elements, like bends, musical directions etc.  By the way, I couldn't get the 64 bit version to work on my 64 bit windows machine.    At first I got a message saying it couldn't find msvcp140_1.dll... so I copied that dll into my PATH (env variable) and then I got another generic error saying it couldn't run and to reinstall.    x86 worked great though, so that's what i'll use.

**Reply by cameronwhite** [reply]
Great to hear that you like the new editing workflow!

For the 64-bit issue, did it install successfully but just didn't run? It _should_ be installing that msvc runtime dll automatically during the install if it's required..
If you have a chance to try fully uninstalling powertabeditor and then re-installing the 64-bit version to see if that works, that would also be helpful!

**Reply by BigHonkinDaddy** [reply]
I uninstalled x86 version and reinstalled the 64-bit version.  It installed fine, but when attempting launch got the error.  I'm not sure this error has anything to do with the msvc file... the first time i tried to install it said that file was missing (when in fact it was already on my pc, just in a different folder than it was expecting, apparently.) At that time, I copied the msvc file into the PTE directory as well as to my System32 directory (which is listed in my PATH env variable).  Since then I haven't gotten an error about a missing msvc file...  instead, I now just get a generic "app was unable to start" message.   Here are some screen shots to show you.  if no one else is reporting an issue, don't sweat it...i can easily use the x86 version.

![PTE64bit install success](https://user-images.githubusercontent.com/77425146/132904870-b00eb930-6fd5-4b1e-ab87-cd6577723523.png)
![PTE64bit launch failure](https://user-images.githubusercontent.com/77425146/132904882-994013b4-602e-4716-b7bb-5a3577fe2a73.png)

**Reply by cameronwhite** [reply]
OK thanks! I might see if I can set up a VM to test this out - which Windows version are you on?

**Reply by BigHonkinDaddy** [reply]
Windows 7.  I have another machine, window 10, that i'll try for you.

**Reply by BigHonkinDaddy** [reply]
It worked fine on my Win 10 pc.

**Reply by cameronwhite** [reply]
Interesting, thanks! Likely something specific to Windows 7 , then

**Comment by LuciferianOrbit** [comment]
I installed the most recent x64 version and the first tab I opened in the program, which was created in PTE 1.7, was missing the tempo, repeats, and capo info. The tempo changes and repeats do show up about halfway through the piece. Also, the pitch sounds correct for the capo position, but there is no marker or mention of the capo at the top of the piece. Also, I'd say about half of the time signatures actually show up on the tab even though they seem to all count out just fine. I'm assuming these are all just issues of cross compatibility and formatting. 
The main thing I was hoping this update would solve was the way the player handles tempo changes. There is always a hiccup when changing from one tempo to the next like the first note of the new tempo has the note length of the last tempo or it cuts the last note of one tempo section short. This seems to still be an issue with this version as well.

**Reply by cameronwhite** [reply]
Could you submit an issue (https://github.com/powertab/powertabeditor/issues/new/choose) for the tempo change issue, maybe with a simple file that can reproduce it?

The other issues also sound like bugs that could be fixed, but if that tab can't be posted publicly let me know!

**Comment by Ebdim9th** [comment]
I'm trying unsuccessfully to install on a new laptop; this is what I get when it wont, the second is what I get when it will, depending on the version ,64x, 86x, but won't work when I try to change the tunings:

![Screenshot 2021-11-02 022835](https://user-images.githubusercontent.com/55242755/139804180-1efebb72-3148-450d-b995-441a3ca72258.png)


![Screenshot 2021-11-02 022427](https://user-images.githubusercontent.com/55242755/139804166-b7fc207f-7ec5-47dd-b657-0ab3038167a8.png)


Re-installing and repairing don't work.

**Reply by cameronwhite** [reply]
This was filed as #339

**Reply by Ebdim9th** [reply]
right on, thanks!
